src/System.Diagnostics.Process/tests/ProcessTestBase.cs

```diff
@@ -100,6 +100,16 @@ protected void StartSleepKillWait(Process p)
         /// <param name="program"></param>
         /// <returns></returns>
         protected static bool IsProgramInstalled(string program)
+        {
+            return !(GetProgramPath(program) is null);
```

Comment:
We would normally just do "!=null"

Reply:
Who knows if we'll have some `isnot null` form :smirk: `is object` works but I agree with Jan that's ugly.